lib/which_update.rb

```diff
@@ -43,10 +43,19 @@ def stats(source: nil)
       nil
     end
 
-    def update_and_save!(source: nil, commit: false)
+    def update_and_save!(source: nil, commit: false, install_missing: false)
       source ||= default_source
       db = ExecutablesDB.new source
       db.update!
+
+      if install_missing
+        (Formula.core_names - db.formula_names).each do |formula|
+          ohai "Installing #{formula}"
+          system HOMEBREW_BREW_FILE, "install", "--formula", formula
+        end
+        db.update!
+      end
```

Comment:
How much disk space does this use?

Reply:
Depends on how many are missing tbh. Currently, there are 55. The idea is that there will only ever be a handful when this runs (i.e. any new formulae added that day) so it won't be too big a deal to install them all.

Reply:
Ok I misread how this work. Do we ever update the list of existing formulae? The binaries shipped can change from version to version.

Reply:
That's the issue I've brought up above. Currently no.